tasklib/serializing.py

```diff
@@ -173,12 +173,14 @@ def deserialize_annotations(self, data):
         return [TaskAnnotation(self, d) for d in data] if data else []
 
     def serialize_tags(self, tags):
-        return ','.join(tags) if tags else ''
+        return list(tags)
```

Comment:
The point of the serialization is to turn the object into a string representation, so this should be cast as a string.

That said, I am not yet entirely convinced this would work (i.e. with older versions of the TW). Maybe we can revive travis @robgolding ?

Reply:
I have been using this branch for weeks with the latest taskwarrior version. The only issue that arose (#83) was due to me not knowing python well and is now fixed.
Why would we need to support old taskwarrior versions with new releases?

Reviving travis sounds good anyways though.